x-pack/plugins/ml/public/application/routing/routes/explorer.tsx

```diff
@@ -260,27 +268,31 @@ const ExplorerUrlStateManager: FC<ExplorerUrlStateManagerProps> = ({ jobsWithTim
           </EuiFlexItem>
         </EuiFlexGroup>
       </MlPageHeader>
-      <AnomalyExplorerContext.Provider value={anomalyExplorerContext}>
-        {jobsWithTimeRange.length === 0 ? (
-          <AnomalyDetectionEmptyState />
-        ) : (
-          <Explorer
-            {...{
-              explorerState,
-              overallSwimlaneData,
-              showCharts,
-              severity: tableSeverity.val,
-              stoppedPartitions,
-              invalidTimeRangeError,
-              selectedJobsRunning,
-              timeBuckets,
-              timefilter,
-              selectedCells,
-              swimLaneSeverity,
-            }}
-          />
-        )}
-      </AnomalyExplorerContext.Provider>
+      <StyledComponentsThemeProvider>
+        <CasesContext owner={[]} permissions={casesPermissions!}>
```

Comment:
@jonathan-buttner what should I pass as an `owner` prop?

Reply:
What you have is correct actually. When the `owner` is an empty array we show the owner selector which is the behavior you want.

https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/create/form.tsx#L81